connect/runtime/src/test/java/org/apache/kafka/connect/runtime/isolation/PluginUtilsTest.java

```diff
@@ -158,6 +159,21 @@ public void testPluginUrlsWithJars() throws Exception {
         assertUrls(expectedUrls, PluginUtils.pluginUrls(pluginPath));
     }
 
+    @Test
+    public void testOrderOfPluginUrlsWithJars() throws Exception {
+        createBasicDirectoryLayout();
+        // Here this method is just used to create the files. The result is not used.
+        createBasicExpectedUrls();
+
+        List<Path> actual = PluginUtils.pluginUrls(pluginPath);
+        // simple-transform.jar is created first. In most systems, without sorting within the
```

Comment:
Can you clarify this comment? It sounds like you are saying that the test case may pass in some cases even without the fix in this patch. Is there any way to make it more reliable?

Reply:
I tried to rephrase it (it had a typo too). 

The issue is that [DirectoryStream](https://docs.oracle.com/javase/8/docs/api/java/nio/file/DirectoryStream.html) does not return a directory's listing in any certain order. So there's no easy way to enforce such order in this test case always. But if we don't sort within the `PluginUtils` it will often fail. I feel that's the best I can do now. LMK what you think